lib/ansible/modules/cloud/azure/azure_rm_rediscache.py

```diff
@@ -131,6 +131,13 @@
                 choices:
                     - primary
                     - secondary
+    wait_for_running:
```

Comment:
maybe rename it to "wait_for_completion"? I think wait_for_running is not really clear.

Reply:
or "wait_until_provisioning_complete", sounds a kind of long but is the most clear.....

Reply:
also i think default should be "yes" as it's the case for most of the resources

Reply:
i don't think default should be yes. this is long running operation, it's not good to default ask user waiting long time.

Reply:
or as Matt suggested `wait_for_provisioning`?